flow/core/rewards.py

```diff
@@ -441,3 +441,35 @@ def miles_per_gallon(env, veh_ids=None, gain=.001):
     mpg /= 1609.0
 
     return mpg * gain
+
+def instantaneous_power(env, veh_ids=None, gain=.001):
+    """Calculate the instantaneous power for every simulation step specific to the vehicle type.
+
+    The energy model used is based on the vehicle type:
+    veh_id = "Prius"  --> energy model is PriusEnergy in flow.core.kernel.toyata_energy
+    veh_id = "Tacoma" --> energy model is TacomaEnergy in flow.core.kernel.toyata_energy
+    veh_id = "EV"     --> energy model is PDMElectric in flow.core.kernel.power_demand
+    veh_id = "human"  --> energy model is PDMCombustionEngine in flow.core.kernel.power_demand
```

Comment:
update the docstring accordingly

Reply:
Change `veh_ids` in the function header to `veh_id`. This is so we can match the getter functions in the flow/core/kernel/vehicle/traci.py . If you could make the changes to the `miles_per_gallon` function in rewards.py, that would be awesome